core/domain/exp_jobs_one_off.py

```diff
@@ -879,7 +879,7 @@ def map(file_model):
                 yield (WRONG_INSTANCE_ID, instance_id)
             else:
                 filename = catched_groups.group(2)
-                filepath = 'assets/' + filename
+                filepath = 'assets/image/' + filename
```

Comment:
As an FYI I think we can just go ahead and delete this entire job. It's already been run in production.

Reply:
Thanks Sean, just to confirm, do you mean we don't need to run any of the one off jobs defined here in the future?

Reply:
Thanks Sean, just to confirm, do you mean we don't need any of the one off jobs defined here in the future?

Reply:
We don't need to run DeleteImagesFromGAEJob, ValidationOfImagesOnGCSJobUsingExps or ImageDataMigrationJob in the future; the other jobs may still be needed. But these three jobs (and the tests for them) can be deleted, I think. Not sure if that means that the constant at the top is now unused, but if so we can delete it too.

Thanks!